test/functional/commands/recordscreen-e2e-specs.js

```diff
@@ -0,0 +1,148 @@
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import AndroidDriver from '../../..';
+import sampleApps from 'sample-apps';
```

Comment:
This is the source of the CI failure. This package no longer uses `sample-apps`, importing `android-apidemos` directly instead. If you update your dependencies (`rm -rf node_modules && npm install`) you will see the same error.

It is best to import `DEFAULT_CAPS` from https://github.com/appium/appium-android-driver/blob/master/test/functional/desired.js

See https://github.com/appium/appium-android-driver/blob/master/test/functional/commands/element-e2e-specs.js#L5 for an example.